Client/Frontend/Browser/URLBarView.swift

```diff
@@ -182,7 +182,11 @@ class URLBarView: UIView, BrowserLocationViewDelegate, UITextFieldDelegate {
     }
 
     func textFieldDidBeginEditing(textField: UITextField) {
-        textField.selectAll(nil)
+        // Without the async dispatch below, the selectAll() doesn't work
+        // intermittantly, and also crashes on iPhone 6 Plus (bug 1124310).
+        dispatch_async(dispatch_get_main_queue(), {
+            textField.selectAll(nil)
```

Comment:
Yeah, I agree that <code>selectAll</code> should probably be reserved for the context menu...that <code>sender</code> argument was throwing me off. But where do you see that it's not supported in Swift?